DP02_04b_Intermediate_Butler_Queries.ipynb

```diff
@@ -9,8 +9,8 @@
     "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\" alt=\"Rubin Observatory logo, a graphical representation of turning stars into data.\">\n",
```

Comment:
<p>Might be worth changing "the DP0.2 collection" to like "the DP0.2 collection, which is named <code>2.2i/runs/DP0.2</code>".  Just to help clarify that there is no magical query syntax involved in the collection name in the code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Good point!</p>